services/deployment.md

```diff
@@ -1,9 +1,25 @@
 # Deployment
 
+## Sequence of deployment events
+* ask pulbot to deploy
+* it sends a deployment event to github
+* github POSTs that event to heaven via a hook set up in the pulibrary organization
+* heaven catches the event and runs the deployment
+
+## Things that could go wrong:
+* If pulbot is down it won't receive the event. It used to ack messages but at some point we updated and it doesn't do that anymore
+  * troubleshoot: `pulbot ping` it should pong.
+  * pulbot is on lib-ruby-dev
+* go to gh pulibrary org > webhooks > you can see al lthe events that have been fired recently.
```

Comment:
Can we just put a link to where the recent events are listed?  Also, typo: `al lthe`

Reply:
👍 fixed